generated_code/general.py

```diff
@@ -0,0 +1,14 @@
+from yateto import Tensor
+from yateto.input import parseXMLMatrixFile, parseJSONMatrixFile
+
+def addStiffnessTensor(generator):
+    stiffnessTensor = Tensor('stiffnessTensor', (3, 3, 3, 3))
+    direction = Tensor('direction', (3,))
+    christoffel = Tensor('christoffel', (3,3))
+
+    computeChristoffel = christoffel['ik'] <= stiffnessTensor['ijkl'] * direction['j'] * direction['l']
+    generator.add('computeChristoffel', computeChristoffel)
+
+def includeMatrices(matricesDir):
+    matrices = parseJSONMatrixFile('{}/sampling_directions.json'.format(matricesDir))
+    return set([matrices.samplingDirections])
```

Comment:
This is anisotropy related, do we need it for all equations now?!

Reply:
It probably won't hurt _too_ much to have it in... But I get it—I'll disable it if we don't have anisotropy enabled.